src/lib/queries/getMonthlyTotals.ts

```diff
@@ -48,13 +48,17 @@ export default async function getMonthlyTotals(
     );
   });
 
-  (accounts.root?.childrenIds || []).map(
-    (childId: string) => aggregateChildrenTotals(
-      accounts[childId],
-      accounts,
-      todayQuotes,
-      monthlyTotals,
-    ),
+  accounts.root?.childrenIds.forEach(
+    (childId: string) => {
+      aggregateChildrenTotals(
+        accounts[childId],
+        accounts,
+        todayQuotes,
+        monthlyTotals,
+      );
+
+      monthlyTotals[accounts[childId].type.toLowerCase()] = monthlyTotals[childId];
```

Comment:
same as in `getAccounts`, now we can access the first level asset, expense accounts with `monthlyTotals.asset`, `monthlyTotals.expense`, etc.